docs/examples/example_satellite_observations.ipynb

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    dKE.plot(ax=ax[0,0], cmap=ke_cmap, \
</pre><p>You actually don't need the <code>\</code> here :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocean-eddy-cpt/gcm-filters/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>